.travis/.travis.test-common.sh

```diff
@@ -114,17 +114,17 @@ testCreateCluster1() {
 testScaleCluster() {
   info
   if [ "$CRD" = "1" ]; then
-    os::cmd::expect_success_and_text '${BIN} patch sparkcluster my-spark-cluster -p "{\"spec\":{\"worker\": {\"instances\": 1}}}" --type=merge' '"?my-spark-cluster"? patched' || errorLogs
+    os::cmd::expect_success_and_text '${BIN} patch SparkCluster my-spark-cluster -p "{\"spec\":{\"worker\": {\"instances\": 1}}}" --type=merge' '"?my-spark-cluster"? patched' || errorLogs
```

Comment:
in this context it's case insensitive (`oc patch SeRvIcE foo` also works)

Reply:
there are more yamls in the `./examples`. Like this one https://github.com/radanalyticsio/spark-operator/blob/master/examples/cluster.yaml

Reply:
cool, i'll revert this to lowercase